tests/e2e/cairo1/test_cairo1_testing.py

```diff
@@ -138,3 +140,32 @@ def test_report_slowest(protostar: ProtostarFixture, copy_fixture: CopyFixture):
         ignore_exit_code=True,
     )
     assert "Slowest test cases" in result
+
+
+def test_dependencies(protostar: ProtostarFixture, copy_fixture: CopyFixture):
+    copy_fixture("cairo1_project", "./cairo1_project")
+    copy_fixture(
+        "cairo1/test_with_deps.cairo", "./cairo1_project/tests/test_with_deps.cairo"
+    )
+    os.chdir("./cairo1_project")
+
+    result = protostar(["--no-color", "test-cairo1", "tests"])
+
+    assert result
```

Comment:
please make an assertion about the test results

Reply:
Right, I missed that, thanks for pointing it out!

Reply:
[add missing asserts](https://github.com/software-mansion/protostar/pull/1601/commits/4a800c15cb76160c262570f9b5cc02700389fdec)